src/gwt/src/org/rstudio/studio/client/workbench/addins/Addins.java

```diff
@@ -64,7 +74,8 @@ public final static RAddin decode(String decoded)
                splat[1],
                splat[2],
                splat[3],
-               splat[4]);
+               splat[4] == "false" ? false : true,
```

Comment:
Got it!

On Mon, Dec 14, 2015 at 5:29 PM, Kevin Ushey <notifications@github.com>
wrote:

> In src/gwt/src/org/rstudio/studio/client/workbench/addins/Addins.java
> <https://github.com/rstudio/rstudio/pull/538#discussion_r47568917>:
>
> > @@ -64,7 +74,8 @@ public final static RAddin decode(String decoded)
> >                 splat[1],
> >                 splat[2],
> >                 splat[3],
> > -               splat[4]);
> > +               splat[4] == "false" ? false : true,
>
> Part of the reason for this was to ensure that we would default to true
> in case the Interactive field were misspecified.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rstudio/pull/538/files#r47568917>.
>